lib/server.js

```diff
@@ -39,6 +39,7 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
                 Config.client.variableName,
                 ' = %s;</script>'
               ].join('');
+var TEMPLATE_REDIRECT = '<html><head><meta http-equiv="refresh" content="0;URL=\'%s\'" /></head></html>';
```

Comment:
The redirection thing may be a fundamental issue with the nature of react-engine as a view engine rather than a routing engine. I'm still thinking middleware as an alternative approach. An [internal redirect](https://github.com/paypal/react-engine/pull/95/files#diff-c945a46d13b34fcaff544d966cffcabaR90) like @chunkiat82 proposed actually seems more attractive from a "seamless" perspective, but unfortunately one problem with that approach is that the model may not contain the correct props to render for the redirected route. 😫